automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/security/AuthDomainSpec.scala

```diff
@@ -28,169 +30,468 @@ class AuthDomainSpec extends FreeSpec /*with ParallelTestExecution*/ with Matche
 
 
   "A workspace with an authorization domain" - {
+    "with one group inside of it" - {
+      "can be created" in withWebDriver { implicit driver =>
+        withGroup("AuthDomainSpec") { authDomainName =>
+          withCleanUp {
+            val workspaceListPage = signIn(Config.Users.fred)
 
-    "can be created by a user who is in a managed group" in withWebDriver { implicit driver =>
-      withGroup("AuthDomainSpec") { authDomainName =>
-        withCleanUp {
-          val workspaceListPage = signIn(Config.Users.fred)
-
-          val workspaceName = "AuthDomainSpec_create_" + randomUuid
-          register cleanUp api.workspaces.delete(projectName, workspaceName)
-          val workspaceDetailPage = workspaceListPage.createWorkspace(projectName, workspaceName, Option(authDomainName)).awaitLoaded()
+            val workspaceName = "AuthDomainSpec_create_" + randomUuid
+            register cleanUp api.workspaces.delete(projectName, workspaceName)
+            val workspaceDetailPage = workspaceListPage.createWorkspace(projectName, workspaceName, Set(authDomainName)).awaitLoaded()
```

Comment:
definitely. that name came from pre-existing code so i'll see if there are other areas to change along with it